src/bio.c

```diff
@@ -96,7 +74,6 @@ static size_t bioWorkerNum(const bio_worker_data *const bwd) {
     return (size_t)(bwd - bio_workers);
 }
 
-static unsigned long bio_jobs_counter[BIO_NUM_OPS] = {0};
```

Comment:
Unfortunately, here we have an issue.

There are 5 job types.  There are 4 workers.  The code is keeping a counter for each job type, but the mutex queue is for each worker.

I think we need to leave this for backwards compatibility.